azure-pipelines.yml

```diff
@@ -127,3 +127,18 @@ jobs:
   - script: echo $(NODE_OPTIONS)
   - script: "scripts/swagger-to-sdk.sh Azure/$(AZURE_SDK_REPO) -v $(AZURE_SDK_PARAMS)"
     displayName: "Swagger to SDK script"
+
+- job: "Spellcheck"
+  condition: "not(variables['PRIVATE'])"
```

Comment:
I copied this from the Avocado task, but maybe it should be run on your internal side too